CMakeLists.txt

```diff
@@ -25,6 +25,15 @@ endif()
 
 include(AddSYCLExecutable)
 
+# ------------------
+# Extensive mode for improved coverage
+option(SYCL_CTS_ENABLE_EXTENSIVE_MODE
+       "Enable extensive mode with extended type coverage" OFF)
```

Comment:
```suggestion
       "Enable extensive mode with extended type coverage" ON)
```

Reply:
@ProGTX Please, review the latest changes at 9b95609 and d773e0a

Reply:
Thank you for the changes!